src/main/java/thePackmaster/cards/thieverypack/AbstractThieveryCard.java

```diff
@@ -0,0 +1,18 @@
+package thePackmaster.cards.thieverypack;
+
+import thePackmaster.cards.AbstractPackmasterCard;
+
+public abstract class AbstractThieveryCard extends AbstractPackmasterCard {
+	public AbstractThieveryCard(String ID, int cost, CardType type, CardRarity rarity, CardTarget target) {
+		super(ID, cost, type, rarity, target);
+		setBackgroundTexture(
+			"anniv5Resources/images/512/thieverypack/" + type.name().toLowerCase() + ".png",
+			"anniv5Resources/images/1024/thieverypack/" + type.name().toLowerCase() + ".png"
+		);
+	}
+
+	@Override
+	public void upp() {
+		// Yay
```

Comment:
Why do this and then use upgrade instead? This way you could've skipped the !upgraded check and the description/name upgrading

Reply:
Yeah sorry, didn't like the name of `upp`, and since I copied my old code, it's better for me to directly use `upgrade` myself. I test the description change on upgrade so that's fine. I can fix my code to use `upp` if you really want though.

Reply:
Nah, just wanted to mention it. All good